app/assets/javascripts/tabs.js

```diff
@@ -1,6 +1,6 @@
 $( document ).ready(function() {
   const url = new URL(window.location);
-
+  $(".lux-tabs-container li").removeClass( "active" );
```

Comment:
A weird thing happens when I press "back" in the browser - the anchor sticks around as #access but on the component. Maybe a fluke?

Reply:
@tpendragon I think I fixed this in my latest push. LMK if that works for you.

Reply:
@sdellis The back button thing still happens for me. https://pulfalight-staging.princeton.edu/catalog/C0159_c000001 Click "access and use" and then hit the back button. Maybe we'll have to just not render those tabs at all unless you're on a collection component?

Reply:
@tpendragon I think I finally got it with this latest commit.🤞  Thanks for catching that.

Reply:
Yup, that seems to work. PR just needs rebased tomorrow and we're good.